msys2-runtime/update-patches.sh

```diff
@@ -19,6 +17,11 @@ die "Clean worktree required"
 git rm 0*.patch ||
 die "Could not remove previous patches"
 
+base_tag=refs/tags/cygwin-"$(sed -ne 'y/./_/' -e 's/^pkgver=//p' <PKGBUILD)"-release
+source_url=$(sed -ne 's/^source=\([^:]\+::\)\?["'\'']\?\([^"'\''#?=&,;[:space:]]\+[^)"'\''#?=&,;[:space:]]\).*/\2/p' <PKGBUILD)
```

Comment:
If you'd rather just paste the `git://` url in here instead of this ridiculous (probably not 100% fool-proof) regex, I could get behind that.